{{ cookiecutter.formal_name }}/Dockerfile

```diff
@@ -11,43 +11,45 @@ ARG PY_VERSION
 ARG SYSTEM_REQUIRES
 
 # Make sure installation of tzdata is non-interactive
-ENV DEBIAN_FRONTEND="noninteractive"
+ENV DEBIAN_FRONTEND="noninteractive" \
+# Disable pip warnings about running as root (pip >= 22.1)
+    PIP_ROOT_USER_ACTION="ignore" \
+# Disable pip progress bar
+    PIP_PROGRESS_BAR="off"
```

Comment:
Isn't this redundant now that we removed `--tty` from the Docker command line?

Reply:
Kind of....

I did some more experimentation. `pip` appears to have two dynamic elements: spinners and progress bars.

The spinners are disabled with a [check](https://github.com/pypa/pip/blob/main/src/pip/_internal/cli/spinners.py#L122) to `isatty()`.

The progress bar, though, is not explicitly disabled in `pip` without using its explicit `progress-bar` option. That said, since `pip` uses `rich` for its progress bar, `rich` [will not print](https://github.com/Textualize/rich/blob/master/rich/console.py#L915) dynamic content to the screen if it isn't running in a terminal.

This results in `rich` only printing the final state of the progress bar....like this for instance:
```
>>> Running Command:
>>>     docker run --volume /home/user/tmp/beeware-tutorial/pyside6test/linux:/app:z --volume /home/user/.briefcase:/home/brutus/.briefcase:z --rm 
briefcase/com.example.pyside6test:py3.10 python3.10 -m pip install --upgrade --no-user 
--target=/app/appimage/pyside6-test/pyside6-test.AppDir/usr/app_packages 'pyside6>=6.2.4'
Collecting pyside6>=6.2.4
  Downloading PySide6-6.2.4-6.2.4-cp36.cp37.cp38.cp39.cp310-abi3-manylinux1_x86_64.whl (268.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 268.2/268.2 MB 7.2 MB/s eta 0:00:00
Collecting shiboken6==6.2.4
  Downloading shiboken6-6.2.4-6.2.4-cp36.cp37.cp38.cp39.cp310-abi3-manylinux1_x86_64.whl (1.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 68.9 MB/s eta 0:00:00
Installing collected packages: shiboken6, pyside6
Successfully installed pyside6-6.2.4 shiboken6-6.2.4
>>> Return code: 0
```

Given this, it's probably better _not_ to pass the `progress-bar=off` option here in the `Dockerfile` and also back in `briefcase/create.py`.